examples/conf/kubewatch.conf.webhook.yaml

```diff
@@ -13,16 +13,18 @@ handler:
   flock:
```

Comment:
this file should be duplicated for google chat, not overwriting the webhook example.